### DISCUSSION TITLE
Record manually payment transactions in the order CP edit

### DISCUSSION THREAD

**Discussion by philipboomy** [craftcms/commerce#3475] [Ideas]
There are times we need to take a manual payment in Stripe dashboard. It would be great to be able to record this payment in Craft Commerce by adding the payment id or payment reference number in the order edit screen. This would then mean it could mark order as paid and we can then see the payment in the payment transaction tab as well refund if needed.

**Comment by andrewhawkes** [comment]
It would be great to have something similar to Shopify where you can just mark an order as "paid" and add notes to the transaction. 
Ideally, it would be good to have a status log of all changes to a commerce order too.
